app/src/main/java/org/simple/clinic/instantsearch/InstantSearchEvent.kt

```diff
@@ -34,3 +34,8 @@ data class BlankBpPassportResultReceived(val blankBpPassportResult: BlankBpPassp
 
   override val analyticsName: String = "Instant Search : Blank BP Passport Result Received"
 }
+
+sealed class BpPassportScanned : InstantSearchEvent() {
+
```

Comment:
Because this is a result that is passed from another screen. This is following the same naming convention that we followed for handling BP passport scan results in the home screen, so used those to make it consistent.